### PR TITLE
Fix Day 2 of the Korean translation

### PR DESCRIPTION
**Pull request by jooyunghan**
This covers 12, 13, 14 in TOC.

### CODE REVIEW

File: po/ko.po [hunk 1]
```diff
@@ -5727,22 +5724,22 @@ msgid ""
 "Click(Click)` with a top level `struct Click { ... }`. The inlined version "
 "cannot implement traits, for example."
 msgstr ""
-"* 위의 예제에서 `keyPress`의 `char`, `Cilck`의 `x`와 `y`의 접근은 `match`구"
+"* 위의 예제에서 `KeyPress`의 `char`, `Cilck`의 `x`와 `y`의 접근은 `match`구"
 "문   안에서만 동작 합니다.\n"
-"* `match`는 열거형에서 숨겨진 판별필드까지 검사합니다.\n"
-"* `WebEvent::Click { ... }`은 최상위 레벨 구조체 `Click {...}`이 있는   "
```
Comment (jooyunghan) [inline po/ko.po:5733]: Other fixes are minor, but this is not, IMO. The previous translation was too vague. I couldn't even understand what it says. Probably, the original English text was not clear as well. ;-)

Jiyong, please review the new translation of this part.

Comment (jiyongp) [reply po/ko.po:5733]: `match`는 주어진 열거형 타입이 실제로 어떤 variant인지 판단하기 위해, 그 variant의 종류가 기록된, 숨겨진 필드의 값을 검사합니다.

Comment (jooyunghan) [reply po/ko.po:5733]: Thanks for the suggestion. (I'll use it.)

But, what I was asking is L5733 ;-)

Comment (mgeisler) [reply po/ko.po:5733]: Hey folks, I found problems myself when looking at the Danish translation... So please don't hesitate to send fixes to the English version. The language can definitely be improved.

**Review by jiyongp — APPROVED** [review]

File: po/ko.po [hunk 2]
```diff
@@ -5727,22 +5724,22 @@ msgid ""
...
-"`WebEvent::Click(Click)`와 정확히 같진 않습니다. 예를 들어 인라인   버전에서"
-"는 트레이트를 구현 할 수 없습니다."
+"* `match`는 열거형에서 숨겨진 식별자 필드를 검사합니다.\n"
+"* `WebEvent::Click { ... }`은 최상위 레벨 구조체 `struct Click {...}`를 따로 정의하고"
+" `WebEvent::Click(Click)`처럼 튜플 형태로 정의한 것과 정확히 같진 않습니다. 예를 들어"
+" `WelbEvent::Click { ... }` 로 정의한 경우, 구조체 형태와 유사하지만 트레이트를 구현 할 수 없습니다."
```
[the remark this answers is not in the document]
Comment (jooyunghan) [reply po/ko.po:5733]: Oh, let me fix the typo in a follow up.

**Pull request merged**